### PR TITLE
elasticsearch: ICU plugin test

### PR DESCRIPTION
**Pull request by kaplun**
* Adds a test that checks if ICU is installed.

Signed-off-by: Panos Paparrigopoulos <panos.paparrigopoulos@cern.ch>

### CODE REVIEW

**Comment by jmartinm** [comment]
This branch passes Travis tests, shall we merge it?

File: inspire/testsuite/test_ICU.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+from invenio.testsuite import InvenioTestCase
+
+
+class IcuPluginTests(InvenioTestCase):
+
+    def test_icu(self):
+        from invenio_ext.es import es
+        self.assertTrue('analysis-icu' in es.cat.plugins())
```
Comment (jacquerie) [inline inspire/testsuite/test_ICU.py]: To be more informative, maybe add a `msg='The ICU analyzer plugin is not installed. Please add it by following the instructions at https://github.com/elastic/elasticsearch-analysis-icu'` argument?

Comment (jmartinm) [inline inspire/testsuite/test_ICU.py]: > To be more informative, maybe add a msg='The ICU analyzer plugin is not installed. Please add it by following the instructions at https://github.com/elastic/elasticsearch-analysis-icu' argument?

:+1:

**Comment by Panos512** [comment]
added a note at #528

**Pull request merged**